test.sh

```diff
@@ -4,10 +4,10 @@ env=$1
 file=""
 fails=""
 
-if [[ "${env}" == "stage" ]]; then
-  file="docker-compose-dev.yml"
-elif [[ "${env}" == "dev" ]]; then
+if [[ "${env}" == "dev" ]]; then
```

Comment:
The default `env` is always 'stage' - all tests run the file set as `docker-compose-stage.yml`: https://github.com/apoclyps/my-dev-space/blob/master/.travis.yml#L62

Reply:
@apoclyps Should stage be using `docker-compose-stage.yml`?

Reply:
yes; it should. Although right now there is very little difference between dev and stage docker compose files. The only difference was the dev Dockerfiles had volume mounts (which are not being used anymore). feel free to correct it in this PR or leave it as a follow up.

Reply:
@apoclyps That's what I've aimed to do here, with the order dev -> stage -> prod 👍